mathml/relations/html5-tree/display-1.html

```diff
@@ -56,6 +56,25 @@
               assert_less_than_equal(mspace_inline.right, after_inline.left,
                                      "content after must be on the right of math");
           }, `Test inline math ${transform}`);
+
+          var before_width = getBox("before_width");
+          var mspace_width = getBox("mspace_width");
+          var after_width = getBox("after_width");
+          test(function() {
+              assert_true(MathMLFeatureDetection.has_mspace());
+              assert_approx_equals(before_width.left, content.left, 1,
+                                   "content before must be left aligned");
+              assert_approx_equals((mspace_width.left + mspace_width.right) / 2,
+                                   (content.left + content.right) / 2,
+                                   1,
+                                   "math must be centered.");
+              assert_approx_equals(after_width.left, content.left, 1,
+                                   "content before must be left aligned");
```

Comment:
after*

Reply:
Good catch, since this was copy paste, I fixed the original comment as well.